snuba/cli/consumer.py

```diff
@@ -91,23 +95,24 @@
     help="Runs a stateful consumer (that manages snapshots) instead of a basic one.",
 )
 def consumer(
-    raw_events_topic,
-    replacements_topic,
-    commit_log_topic,
-    control_topic,
-    consumer_group,
-    bootstrap_server,
-    dataset,
-    max_batch_size,
-    max_batch_time_ms,
-    auto_offset_reset,
-    queued_max_messages_kbytes,
-    queued_min_messages,
-    log_level,
-    dogstatsd_host,
-    dogstatsd_port,
-    stateful_consumer,
-):
+    *,
+    raw_events_topic: Optional[str],
+    replacements_topic: Optional[str],
+    commit_log_topic: Optional[str],
+    control_topic: Optional[str],
+    consumer_group: str,
+    bootstrap_server: Sequence[str],
```

Comment:
This has None as default.

Reply:
Removed the unused default: 0133b2dc60cbc37e9d0e53508321150f2f678405